agent/agent.sh

```diff
@@ -1,23 +1,29 @@
 #!/bin/sh
 
-set -e
+function signal {
+  cfn-signal -e $1 --stack $STACK --resource ASG --region $REGION
+  [ $? -ne 0 ] && echo "i2kit agent: cfn-signal failed to be sent."
+  [ $? -eq 0 ] && echo "i2kit agent: cfn-signal successfully sent."
```

Comment:
Does this work? I'd think that at this point $? would be the exit code of comparison not the cfn-signal call.

Reply:
right. i will update